Source/WPEFramework/Config.h

```diff
@@ -709,6 +721,9 @@ namespace PluginHost {
         inline uint16_t IdleTime() const {
             return (_idleTime);
         }
+        inline void SetIdleTime(const uint16_t& newValue)  {
```

Comment:
mmm, confusing because some discussions point to a better performance by using const.  
But I have mainly used them for code readiness to  indicate that parameter shall  not be changed by the functions.
I removed the new ones , but for info, many other files use const for simple types: example: GATTSocket.h, ControllerJsonRpc.cpp...maybe we can clean them up later.

Reply:
Eh think we do not understand each other. It was not about the const. I'll give you a call